src/components/ArticleList.jsx

```diff
@@ -0,0 +1,56 @@
+import { useState, useEffect } from "react";
+import ArticleCard from "./ArticleCard";
+
+function ArticleList({articleSearch, articleFilter}) {
+  const [searchedArticles, setSearchedArticles] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const [error, setError] = useState(null);
+
+  useEffect(() => {
+    setIsLoading(true);
+    let url = [];
+
+    if (!articleSearch && !articleFilter) {
+      url.shift();
+      url.push("/articles/");
+    }
+    if (articleSearch && !articleFilter) {
+      url.shift();
+      url.push(`/articles/?search=${articleSearch}`);
+    }
+    if (!articleSearch && articleFilter) {
+      url.shift();
+      url.push(`/articles/?filter=${articleFilter}`);
+    } else if (articleSearch && articleFilter) {
+      url.shift();
+      url.push(`/articles/?search=${articleSearch}&&?/articles/?filter=football=${articleFilter}`);
+    }
```

Comment:
Currently no search works my side although filtering does - does search work on your side?

Do you have the ability to search for article by name in your backend?

Reply:
Search was not enabled on backend, but I didn't realise this until I had written most of the code. Should I just remove the search box.